plugins/generic/externalFeed/block.tpl

```diff
@@ -1,8 +1,8 @@
 {**
- * plugins/generic/externalFeed/block.tpl
+ * plugins/generic/externalFeed/templates/block.tpl
```

Comment:
It doesn't look like this file actually got moved into `templates`...?

Reply:
this was moved to `templates` then moved back to root folder (where `getBlockTemplateFilename` expects it) but i forgot to modify the path again.